### DISCUSSION TITLE
no matching manifest for windows/amd64

### DISCUSSION THREAD

**Discussion by controlc338** [moby/moby#44071] [Q&A] [answered]
**Step 1/1 : FROM redis:7.0.4-alpine
7.0.4-alpine: Pulling from library/redis
1 error occurred:
        * Status: no matching manifest for windows/amd64 10.0.17763 in the manifest list entries, Code: 1**

**Comment by vvoland** [comment, marked as answer]
The redis:7.0.4-alpine only supports Linux containers and doesn't have any image for Windows containers.
See: https://hub.docker.com/layers/library/redis/7.0.4-alpine/images/sha256-7c9e7c305125c68354c4902ed09a3c6da96436a5a2929b73f29748d56cf8e3c3?context=explore

**Reply by thaJeztah** [reply]
Yes, this image only supports Linux. If your intent is to run Linux containers on your Windows machine, and you're running Docker Desktop, then look at [How do I switch between Windows and Linux containers](https://docs.docker.com/desktop/faqs/windowsfaqs/#how-do-i-switch-between-windows-and-linux-containers) in the documentation.

**Reply by controlc338** [reply]
thank you very much.
